### PR TITLE
update READMEs

### PR DESCRIPTION
**Pull request by peterjah**

### CODE REVIEW

File: packages/sc-project-initializer/README.md [hunk 1]
```diff
@@ -8,14 +8,14 @@ This package is meant to facilitate smart contract development.
...
 
-Simply run the following command:
+Simply run the following command with your project name:
 
 ```shell
 npx @massalabs/sc-project-initializer init <projectName>
 ```
 
-Toolkit documentation is available [here](commands/init/README.md)
+The initializer documentation is available [here](commands/init/README.md)
```
Comment (Thykof) [inline packages/sc-project-initializer/README.md:17]:
```suggestion
The initializer documentation is available [here](commands/init/README.md).
```

File: packages/sc-project-initializer/commands/init/README.md [hunk 2]
```diff
@@ -43,33 +30,30 @@ Prerequisites :
...
-Then deploy your contract with:
-
 ```shell
 npm run deploy
 ```
 
 This command will deploy your smart contract on Massa's network corresponding to the given node.
 
-When you deploy a contract the function `constructor` is called. If you want to deploy more contracts or pass parameter to the constructor function you have to modify `src/deploy.ts`. Even if you want the default behavior it's cool to take a look at it.
+When the deployment transaction is executed onchain, the `constructor` function of your deployer SC `assembly/main.ts` will be called.
```
Comment (Thykof) [inline packages/sc-project-initializer/commands/init/README.md:42]:
```suggestion
When the deployment transaction is executed onchain, the `constructor` function of your smaprt contract `assembly/main.ts` will be called.
```

**Review by Thykof — CHANGES_REQUESTED** [review]

File: packages/sc-project-initializer/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (peterjah) [reply packages/sc-project-initializer/README.md:17]: relative path should work

**Review by Thykof — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -2,7 +2,22 @@
 
 This repository is a monorepo for smart-contract development in AssemblyScript for Massa blockchain.
 
-## Build
+## Packages:
+
+### @massalabs/sc-project-initializer
+
+This package makes your first steps into massa smart-contracts development easy:
```
Comment (longgamma-massa) [inline README.md:9]: Sc-initializer (smart-contract initializer), is a package that should be used as a first step when starting to develop smart contracts on Massa blockchain. It consists of the files necessary for setting up the environment for developing a smart-contract project.

File: README.md [hunk 4]
```diff
@@ -2,7 +2,22 @@
...
+
+- [Installation](./packages/sc-project-initializer/README.md)
+- [Build your first smart-contract](./packages/sc-project-initializer/commands/init/README.md)
+
+### @massalabs/sc-deployer
+
+This package is a tool to deploy your smart-contracts on the Massa network:
```
Comment (longgamma-massa) [inline README.md:16]: Sc-deployer is a tool used for deploying your smart contracts on the Massa network:

File: packages/sc-project-initializer/README.md [hunk 5]
```diff
@@ -8,14 +8,14 @@ This package is meant to facilitate smart contract development.
...
 ```
 
-Toolkit documentation is available [here](commands/init/README.md)
+The initializer documentation is available [here](https://github.com/massalabs/massa-sc-toolkit/blob/main/packages/sc-project-initializer/commands/init/README.md).
 
-## Develop
+## Contribute
 
-You can initialize a project with `npx sc-project-initializer init testDir`.
+For development purpose you can initialize a local project with `npx sc-project-initializer init <projectName>`.
```
Comment (longgamma-massa) [inline packages/sc-project-initializer/README.md:21]: For development purposes, you can initialize a local project with `npx sc-project-initializer init <projectName>`.

File: packages/sc-project-initializer/commands/init/README.md [hunk 6]
```diff
@@ -2,11 +2,7 @@
 
 You now have your own AssemblyScript project setup, with Massa's sdk installed.
 
-You can now run `npm run build` to compile your AssemblyScript files.
-
-By default it will build all files in `assembly/contracts` directory.
-
-To use libraries as `massa-as-sdk` and `@massalabs/as` you need to import the required function, for instance :
+To use libraries as `@massalabs/massa-as-sdk` and `@massalabs/as` you need to import the required function, for instance :
```
Comment (longgamma-massa) [inline packages/sc-project-initializer/commands/init/README.md:5]: To use libraries, like `@massalabs/massa-as-sdk` and `@massalabs/as`, you need to import the required function, for instance :

Comment (longgamma-massa) [reply packages/sc-project-initializer/commands/init/README.md:5]: Just a comment: isn't it possible to import all functions at once, not one by one? If yes and if it's not trivial to devs to know this, maybe we should mention it?

File: packages/sc-project-initializer/commands/init/README.md [hunk 7]
```diff
@@ -43,33 +30,30 @@ Prerequisites :
...
-Then deploy your contract with:
-
 ```shell
 npm run deploy
 ```
 
 This command will deploy your smart contract on Massa's network corresponding to the given node.
 
-When you deploy a contract the function `constructor` is called. If you want to deploy more contracts or pass parameter to the constructor function you have to modify `src/deploy.ts`. Even if you want the default behavior it's cool to take a look at it.
+When the deployment transaction is executed onchain, the `constructor` function of your smart contract `assembly/main.ts` will be called.
```
Comment (longgamma-massa) [inline packages/sc-project-initializer/commands/init/README.md:42]: When the deployment operation is executed on-chain, the `constructor` function of your smart contract `assembly/main.ts` will be called.

File: packages/sc-project-initializer/commands/init/README.md [hunk 8]
```diff
@@ -43,33 +30,30 @@ Prerequisites :
...
+The `constructor` function can be modified for specific needs, and its call arguments edited in the deployment script `src/deploy.ts`.
 
-### ... Run unit tests
+## Unit tests
 
-Check examples in `./assembly/__test__/massa-example.spec.ts`
+A test sample is provided as example here:  `./assembly/__test__/massa-example.spec.ts`
 
-Check the documentation on <https://as-pect.gitbook.io/as-pect>
+Test framework documentation is available here: [as-pect docs](https://as-pect.gitbook.io/as-pect)
```
Comment (longgamma-massa) [inline packages/sc-project-initializer/commands/init/README.md:49]: The test framework documentation is available here: [as-pect docs](https://as-pect.gitbook.io/as-pect)

**Review by longgamma-massa — COMMENTED** [review]
couldn't add fixes as 'suggestions' for some reasons. I left them as comments

**Comment by longgamma-massa** [comment]
I only see the Readme for SC-toolkit, not for sc-initializer, sc-deployer separately in their package readmes. We need those too, even if it's a copy of the section from the sc-toolkit's readme. Let's make it a practice to add readme's to all npmjs packages

File: packages/sc-project-initializer/commands/init/README.md (hunk 6 above)
Comment (peterjah) [reply packages/sc-project-initializer/commands/init/README.md:5]: For some reason, it has been splitted in two separate repos `@massalabs/massa-as-sdk`  `@massalabs/as`. thats mean you need to import both separately. a solution to make it simple would be to reexport stuffs from "as" in "as-sdk", thus only importing the sdk would work

**Comment by peterjah** [comment]
all thoses readmes are in the same repo (toolkit) but there is one readme for each of sc-initializer and sc-deployer

**Pull request merged**